docs/faq.md

```diff
@@ -96,7 +96,7 @@ Both Semgrep and CodeQL use static analysis to find bugs, but there are a few di
 - Semgrep operates directly on source code, whereas CodeQL requires a buildable environment
 - Semgrep is LGPL-2.1 and free to run anywhere; CodeQL is not open source and you must pay to run it on any non-open-source code
 - Semgrep supports autofixes; CodeQL does not.
-- Semgrep focuses on speed and ease of use. Because it doesn’t require a buildable environment, it doesn’t have some of the analysis features like interprocedural dataflow analysis that CodeQL does. (Semgrep does have [limited intraproceedural dataflow](/writing-rules/data-flow/data-flow-overview/)).
+- Semgrep focuses on speed and ease of use. Because it doesn’t require a buildable environment, it doesn’t yet have some of the analysis features that CodeQL does. Semgrep has [limited intraproceedural dataflow](/writing-rules/data-flow/data-flow-overview/) and the proprietary [DeepSemgrep](./deepsemgrep/) extension to Semgrep provides parity in terms of interprocedural dataflow analysis for a subset of supported languages.
```

Comment:
```suggestion
- Semgrep focuses on speed and ease of use. Because Semgrep doesn’t require a buildable environment, it doesn’t yet provide some of the analysis features of CodeQL. Semgrep has [limited intraprocedural dataflow](/writing-rules/data-flow/data-flow-overview/) and the proprietary [DeepSemgrep](/deepsemgrep/) extension to Semgrep provides parity in terms of interprocedural dataflow analysis for a subset of supported languages.
```